model/products.js

```
@@ -14,13 +14,15 @@ function getProduct(product_id) {
-function search(query) {
+// For different databases:
 
-    var q = "SELECT * FROM products WHERE name ILIKE '%" + query + "%' OR description ILIKE '%" + query + "%';";
+// MySQL/PostgreSQL:
+const query = "SELECT * FROM users WHERE id = ? AND name = ?";
+db.query(query, [userId, userName]);
 
-    return db.many(q);
+// With named parameters:
+const query = "SELECT * FROM users WHERE id = :id AND name = :name";
+db.query(query, { id: userId, name: userName });
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)